esrally/track/loader.py

```diff
@@ -191,17 +184,13 @@ def _load_single_track(cfg, track_repository, track_name):
     try:
         track_dir = track_repository.track_dir(track_name)
         reader = TrackFileReader(cfg)
-
         current_track = reader.read(track_name, track_repository.track_file(track_name), track_dir)
-
         track_processor = TrackProcessorRegistry(cfg)
```

Comment:
As the purpose of this class has changed from being a composite track processor to a registry I propose that we rename this variable e.g. to `registry`.